Dockerfile

```diff
@@ -16,14 +16,14 @@ RUN go test  -c -o=bin/ferretdb -trimpath -tags=ferretdb_testcover,ferretdb_tigr
 
 FROM golang:1.19.3
 
-WORKDIR /
-
 COPY --from=build /src/bin/ferretdb /ferretdb
 
-EXPOSE 27017
-
+WORKDIR /
 ENTRYPOINT [ "/ferretdb" ]
-CMD [ "--listen-addr=:27017", "--postgresql-url=postgres://username:password@hostname:5432/ferretdb" ]
```

Comment:
Just double checking, do we have `FERRETDB_POSTGRESQL_URL` set for when we build docker image from Taskfile.yml?

Reply:
We don't; it should be set by the user. That value did not have much sense and served more than an example; it is properly documented now.